packages/cli/index.js

```diff
@@ -98,7 +97,9 @@ program
   .option('-p, --port <port>', 'port for server to listen on (Default is 8080)')
   .option('-s, --site-config <file>', 'specify the site config file (default: site.json)')
   .option('-d, --dev', 'development mode, enabling live & hot reload for frontend source files.')
+  .description('build then serve a website from a directory')
   .action((userSpecifiedRoot, options) => {
+    logger.info(options);
```

Comment:
If you find printing this useful, should we print this in other commands too?

Reply:
Actually nope, since the `dev` mode is already available, this is quite redundant :sweat_smile: Let me revert this!

Reply:
Fixed in [a27bad9](https://github.com/MarkBind/markbind/pull/2218/commits/a27bad98b61f596abe72f9dfc46d580724ddd2cf)!